### PR TITLE
Todo structure 

### PR DESCRIPTION
**Pull request by killy10o10**
1. Set up the project with webpack.
2. I created an index.html file an add empty To-Do List placeholder.
3. The index.html file was set as a template using the HTML Webpack Plugin.
4. I created an index.js file and set an array of some simple to-do tasks (array of objects).
5. Each task object contains three keys:
description [string].
completed [bool].
index: [number].

6. I added a function to iterate over the tasks array and populate an HTML list item element for each task. 
7. On page load, the created list of tasks is rendered dynamically in the UI.
8. I created a main.scss and set rules for the To-Do List. sass must be loaded by Webpack sass-Loader.

### CODE REVIEW

**Comment by omar-labana** [comment]
# **Invalid Code Review Request ⚠️❌**

Hi @killy10o10 

While you made great effort in this project, unfortunately🚩, I cannot proceed to review your code😓.

linters are not passing, kindly fix linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by killy10o10** [comment]
hi @omar-labana  Thanks for the review but I don't see to get any linter errors locally when I ren them. can you please help me out on this one

**Comment by omar-labana** [comment]
@killy10o10 I can see that there seems to be some issues with GitHub Actions, kindly ask for another CRR and the next reviewer should review this.

**Comment by KDlamini** [comment]
> @killy10o10 I can see that there seems to be some issues with GitHub Actions, kindly ask for another CRR and the next reviewer should review this.

@killy10o10, @omar-labana
Noted 👍🏾
I ran all the linter checks locally as well, they do all pass indeed. I'll continue with the review as suggested.

File: package.json [hunk 1]
```diff
@@ -11,14 +11,12 @@
...
   "devDependencies": {
-    "@babel/core": "^7.18.2",
-    "@babel/preset-env": "^7.18.2",
     "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.2.5",
     "css-loader": "^6.7.1",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.26.0",
+    "eslint-webpack-plugin": "^3.1.1",
```
Comment (KDlamini) [inline package.json]: [OPTIONAL] One way you can troubleshoot the linters issue is to re-install all your ESlint and Stylelint packages by doing the following:

1. In your `package.json` delete the Eslint and Stylelint dependencies manually.
image

2. Then, delete the `package-lock.json` file and `node_modules` directory.
3. Then, run `npm install` to re-create the `package-lock.json` and `node_modules` without Eslint and Stylelint dependencies.

4. Then, go to the linter checks details on your pull request.
image

- Copy the Stylelint command from Github and use it to install Stylelint.
image

- Similarly, Copy the Eslint command from Github and use it to install Eslint as well.
image

5. Commit and push changes.

Let me know if it works and Goodluck! 👍🏾

**Review by KDlamini — APPROVED** [review]
### Hi @killy10o10 🙋‍♀️,  

Great work! 💯 
You have done a great job implementing all the project requirements. 🎊
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (killy10o10) [reply package.json]: @KDlamini  this is very helpful thanks for the solution

**Pull request merged**